docs/usage/nodes-management.md

```diff
@@ -0,0 +1,38 @@
+# Nodes Management
+
+Nodes can be managed from the Control Pannel page by clicking on `MANAGE NODES` button under `Actions` section. This will open the **Node Manager** dialog
+
+![Nodes Manager](../_images/nodes_manager.png)
+
+> [!NOTE]
+> In order to include secure devices, you must setup the **SECURITY KEYS** in Z-Wave settings.
+
+## Add a node
+
+1. Select `Inclusion` and press `NEXT`
+2. Select the inclusion mode. We suggest using `Default` or `Smart start`). Then press `NEXT`.
+3. Put your device in `Inclusion mode` (check your device manual for instructions).
+4. If the selected mode is `Default`, the next steps will depend on the device you are including. If it supports S0/S2 security you will be asked to select the security classes to grant and, if the device requires it, the DSK pin code. If the process completes without errors you will see a message saying that the node has been added and which security class was used.
+
+## Remove a node
+
+1. Select `Exclusion` and press `NEXT`.
+2. Put your device in `Exclusion mode` (check your device manual for instructions).
+3. If the process completes without errors you will see a message saying the node has been removed successfully.
+
+## Replace failed node
+
+Like `Remove Failed Node`, this can only succeed if the node to replace:
+
+- is marked as **Dead** or **Asleep**
+- does not respond to a ping
```

Comment:
I don't think this is entirely clear. **Unknown** nodes can also be replaced if they don't respond to a ping.